easybuild/easyconfigs/t/TRF/TRF-4.09-linux64.eb

```diff
@@ -0,0 +1,23 @@
+easyblock = 'Binary'
+
+name = 'TRF'
+version = '4.09'
+versionsuffix = '-linux64'
+
+homepage = 'https://tandem.bu.edu/trf/trf.html'
+description = """Tandem repeats finder: a program to analyze DNA sequences. Legacy version."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+#source_urls = ['https://tandem.bu.edu/trf/trf409.linux64.download.html']
```

Comment:
@RvDijk can this be fixed? looks like this should be `http://tandem.bu.edu/trf/downloads/`